CONTRIBUTING.md

```diff
@@ -81,39 +81,58 @@ cherry-picked into `book-v2` branch.
-|  Plugin Versions | `go.kubebuilder.io/v2.0.0` | Represents the version of an individual plugin, as well as the corresponding scaffolding that it generates. |
+|  KubeBuilder version | `v2.2.0`, `v2.3.0`, `v2.3.1` | Tagged versions of the KubeBuilder project, representing changes to the source code in this repository. See the [releases] page for binary releases. |
+|  Project version |  `"1"`, `"2"`,`"3-alpha"` | Project version defines the scheme of a `PROJECT` configuration file. This version is defined in a `PROJECT` file's `version`. |
+|  Plugin version | `v2.0`, `v2.1` | Represents the version of an individual plugin, as well as the corresponding scaffolding that it generates. This version is defined in a plugin key, ex. `go.kubebuilder.io/v2.0`. See the [design doc][plugins-design-versioning] for more details. |
 
-Note that PROJECT version should only be bumped if a breaking change is introduced in the PROJECT file format itself.  Changes to the Go scaffolding or the KubeBuilder CLI *do not* affect the PROJECT version.
+### Incrementing versions
 
-Similarly, the introduction of a new major version (`(x+1).0.0`) of the Go plugin might only lead to a new minor (`x.(y+1).0`) release of KubeBuilder, since no breaking change is being made to the CLI itself.  It'd only be a breaking change to KubeBuilder if we remove support for an older version of the plugin.
+Project versions should only be increased if a breaking change is introduced in the PROJECT file scheme itself. Changes to the Go scaffolding or the KubeBuilder CLI *do not* affect project version.
 
-For more information on how the release and plugin versions work, see the the [semver](https://semver.org/) documentation.
+Similarly, the introduction of a new major version (`v(X+1).Y`) of the Go plugin might only lead to a new minor (`vX.(Y+1).Z`) release of KubeBuilder, since no breaking change is being made to the CLI itself. It'd only be a breaking change to KubeBuilder if we remove support for an older version of the plugin.
 
-**NOTE:** In the case of the `v1` and `v2` PROJECT version, a corresponding Plugin version is implied and constant -- `v1` implies the `go.kubebuilder.io/v1` Plugin, and similarly for `v2`.  This is for legacy purposes -- no such implication is made with the `v3` PROJECT version.
+For more information on how the release and plugin versions work, see the [semver](https://semver.org/) documentation.
 
-## Introducing changes in the scaffold files
+**NOTE:** schema for project versions `"1"` and `"2"` were defined before the concept of plugins was introduced, so plugin `go.kubebuilder.io/v2.0` is implicitly used for those project types. Schema for project versions `"3-alpha"` and beyond define a `layout` key that informs the plugin system of which plugin to use.
```

Comment:
`plugin `go.kubebuilder.io/v2.0` is implicitly used for those project types` -> It is not true since it ONLY represents the layout of the files scaffolded by default in V2 projects. V1 projects have a different layout and following the same analogy would be `go.kubebuilder.io/v1.0` besides its code source no longer exist in the project.

Reply:
That is incorrect, they both use(d) `go.kubebuilder.io/v2.0.0`.

Reply:
Ah yes there was a `v1.0.0` plugin. We should just remove discussion of project version 1 to avoid confusion, since we don't talk about it anywhere else.

Reply:
Agreed .. Remove V1 from here as well.